README.rst

```diff
@@ -22,7 +22,7 @@ Pipenv: Sacred Marriage of Pipfile, Pip, & Virtualenv
 ---------------
 
 
-**Pipenv** harnesses `Pipfile <https://github.com/pypa/pipfile>`_, `Pip <https://github.com/pypa/pip>`_, and `Virtualenv <https://github.com/pypa/virtualenv>`_ together in unison to create a single, high-quality tool that is optimized for workflow efficiency and best practices. 
+**Pipenv** harnesses `Pipfile <https://github.com/pypa/pipfile>`_, `Pip <https://github.com/pypa/pip>`_, and `Virtualenv <https://github.com/pypa/virtualenv>`_ together in unison to create a single, high-quality tool that is optimized for workflow efficiency and best practices.
```

Comment:
Could we move doc cleanup into another PR so we have things properly partitioned? That way if this is ever reverted for some reason we're not also reintroducing issues to the documentation.